packages/rlc-codegen/src/interfaces.ts

```diff
@@ -11,8 +11,13 @@ export interface RLCModel {
   responses?: OperationResponse[];
   importSet?: Map<ImportKind, Set<string>>;
   pageInfo?: PageInfo;
+  uriInfo?: UriInfo
 }
 
+export interface UriInfo {
```

Comment:
if possible I would recommend not to use Uri but use Url instead

Reply:
updated.